src/nvim/eval.c

```diff
@@ -8660,24 +8660,17 @@ void eval_fmt_source_name_line(char *buf, size_t bufsize)
   }
 }
 
-void invoke_prompt_callback(void)
+/// Returns the current user input to the prompt in prompt buffer 'buf'
+///         if 'buf' is not a prompt buffer then returns NULL.
```

Comment:
single-quotes usually mean "option". avoid single quotes. These days backticks are usually used for symbols.

```suggestion
/// Gets the current user-input in prompt buffer `buf`, or NULL if buffer is not a prompt buffer.
```